### DISCUSSION TITLE
演算法鬼斧神工，AI 畫作的神來之筆

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#400] [科技新知]
「AI畫作的創造力和想像力讓人驚訝，充滿視覺衝擊力很難想像是AI幾小時就完成，更重要的是，AI繪畫工具Disco Diffusion甚至不需使用者有任何AI知識，僅需輸入幾段描述文字，一幅幅精美的畫作就會以演算法生成。」

看到文章內的畫作實在很難想像是Disco Diffusion AI製作出來的圖畫。僅需做一些基本參數設定以及畫作描述即可。但文中也提到可能會有畫家因此失業的問題，也值得探討

資料來源：https://technews.tw/2022/05/15/ai-disco-diffusion/

**Comment by AngusGuei** [comment]
我一直覺得可能藝術給人的感受也能轉化成電子訊號，因此若能分析定解釋這些訊號，以後像是音樂或是書畫都能用 AI 產出個人喜愛的風格作品

**Comment by YuHengChen54** [comment]
那些畫看起來都有種科幻感欸，看起來很像未來的世界

**Comment by brian431** [comment]
隨著人工智慧的發展，各個產業的人都需要好好思考如何增加自己的不可替代性，以免淪為科技進步下的犧牲品。

**Comment by boson13579** [comment]
這個程式可說是運用了大量的ai運算，先運用自然語言處理，提取關鍵訊息，在利用這些關鍵訊息當參數放進畫作生成的ai運算，可喂是當前ai運算的一個整合

**Comment by lx3211** [comment]
但同時這樣技術也有許多爭議，使用畫師的名字就可以畫出該畫家風格的畫作，導致許多畫家對這項技術相當反彈。

**Comment by Chenszuhong** [comment]
這好像Apple收購的那家音樂公司，可以把心跳變成音樂

**Comment by BingRong1217** [comment]
可能也會有著作權的問題產生

**Comment by skyling0299** [comment]
AI是否具有藝術創作力也成了問題，網路上也不泛各式討論。當然也有著作權歸屬的問題

**Comment by awkward-willy** [comment]
其實只要能讓人們覺得美、感到愉快或啟發等的都可以稱之為藝術吧，不必要分是由誰創作的。只是畫家的畫被當作訓練集的一部分進而延伸出出著作權等爭議以及相關究責問題才是真正難以處理且急需我們規範的。

**Comment by Ben6uo** [comment]
這種技術可以讓電腦程式自動生成圖像或者音樂等作品

**Comment by limitx0** [comment]
AI畫出來的圖有些還真的很好看
雖然可能有些細節沒有處理好

**Comment by Aerofe** [comment]
AI畫出來的圖，其實也蠻具特色的，有點未來的風格

**Comment by awehao** [comment]
感覺可以弄出另一個人工智慧來判斷一幅畫是不是人畫的了

**Comment by larry40305** [comment]
AI的訓練資料必須要由法律來詳細規定，不然會不尊重原創作者
